packages/common/src/utils/filters.ts

```diff
@@ -304,17 +304,23 @@ export const applyDefaultTileTargets = (
     return filterRule;
 };
 
-export const createDashboardFilterRuleFromField = (
-    field: FilterableField,
-    availableTileFilters: Record<string, FilterableField[] | undefined>,
-    includeDefaultValue: boolean,
-    isTemporary: boolean,
-): DashboardFilterRule =>
+export const createDashboardFilterRuleFromField = ({
+    field,
+    availableTileFilters,
+    isTemporary,
+    value,
+}: {
+    field: FilterableField;
+    availableTileFilters: Record<string, FilterableField[] | undefined>;
+    isTemporary: boolean;
+    value?: any;
```

Comment:
I suggest using `unknown` which is "more safe"
```suggestion
    value?: unknown;
```

Reply:
done!